### DISCUSSION TITLE
Welcome to PID_integration Discussions!

### DISCUSSION THREAD

**Discussion by dvejsada** [dvejsada/PID_integration#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by tomassvec** [comment]
Dovoluji si nabídnout jako příklad použití vlastní kartu do HA. Určitě to jde i lépe, ale mě to takto stačí.
```
type: custom:html-template-card
ignore_line_breaks: true
content: |
  <table>
    {% for state in states.sensor if state.name.startswith("Všenory  departure") %}
      {% if loop.index < 5 %}
          <tr>
            <td style='font-size: 90%;'>{{ as_datetime(state.attributes.arrival_timestamp.scheduled).strftime('%H:%M') }}</td>
            <td style='font-size: 90%;'>{{state.attributes.trip.headsign}}</td>
            <td style='font-size: 90%;'>{%- if state.attributes.delay.is_available %}{% if state.attributes.delay.minutes>0 %}<span style='color:red;'>+{{state.attributes.delay.minutes}}m</span>{% else %}<b style='color: green;'>(včas)</b>{% endif %}{% endif -%}</td>
            <td style='padding: 0 0px 0 5px; font-size: 80%; color: #808080;'>{%- if state.attributes.last_stop.name!=None %}{{state.attributes.last_stop.name}}{% endif -%}</td>
          </tr>
      {% endif -%}
    {%- endfor %}
  </table>
```

**Comment by lodyh** [comment]
Hi, love the integration. But for the love of god, I am unable to use the departures that are anytime after the immediate ones. I understand those were saved in a calendar, but I cannot figure out how to pull the data from there. Would you have an example of a flex card or any other card, where you have multiple departures for one stop? I want to view 2 to 3 departures, ie. one in 5 minutes, another in 10 and another in 15. Diky!

**Reply by dvejsada** [reply]
Hi, when setting up the integration (adding new stop), it asks you to fill in how many departures you want to include. Theferore, if you fill in 3, the integration creates 6 sensors (3 for departure times and 3 for corresponding line numbers). For displaying on the dashboard, you can use the flex card example included in the repo (which will show all departures for the stop).

**Comment by ac-vellux** [comment]
Nevím, kde přesně poděkovat, tak to napíšu sem: moc díky za fungující integraci. Dělá přesně, co slibuje, a je snadná na nastavení. Příklad flex table karty je super bonus, který pomohl při formátování. Díky!
